runtime/doc/gui.txt

```diff
@@ -20,8 +20,9 @@ features, whereas help tags with the "ui-" prefix refer to the |ui-protocol|.
 Nvim provides a default, builtin UI (the |TUI|), but there are many other
 (third-party) GUIs that you can use instead:
 
-- Firenvim (Nvim in your web browser!) https://github.com/glacambre/firenvim
+							*vscode*
```

Comment:
Why add this header? It doesn't look like the list below is `vscode` exclusive.

Reply:
It's a common search on https://neovim.io that currently yields no results.

For now this is just a "better than nothing" stub, but can revisit it later, or add more tags for the other items here, etc.